manifests/plugin/foreman_wreckingball.pp

```diff
@@ -0,0 +1,5 @@
+# Installs foreman_wreckingball plugin
```

Comment:
Looks like you included https://github.com/theforeman/puppet-foreman/pull/647 here. For the changelog it'd be nice to separate the two.